src/app/public/shared/input/input.component.ts

```diff
@@ -0,0 +1,20 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { FormControl, FormGroup } from '@angular/forms';
+
+@Component({
+  selector: 'app-input',
+  templateUrl: './input.component.html',
+  styleUrls: ['./input.component.css'],
+})
+export class InputComponent implements OnInit {
+  constructor() {}
+  @Input() inputId = '';
+  @Input() control = new FormControl();
+  @Input() label = '';
+
+  ngOnInit(): void {}
```

Comment:
avoid empty ngoininit

Reply:
changed in https://github.com/MariaAlba1306/github/pull/4